Rise-Vision/common-template
# Don't fail template uptime if Components' uptime fails

## Description
Don't fail template uptime if Components' uptime fails

## Motivation and Context
It has been discussed and we decided to remove the link between Component and Template uptimes

## How Has This Been Tested?
Tested using Charles proxy via the Local Chrome OS Player.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
**Manual Test**  As per the above
**Automated Test** Unit tests were added for the change
**Monitoring** Will monitor that no errors are occurring
**Rollback** Rollback can be done by reverting the PR and following the deployment process
**Documentation** Documentation updated in the uptime calculations: https://docs.google.com/document/d/1PoCrt9np3stm6C1RbaypjZPu1_fvC9qNPiyb134XwnM/edit?ts=5d447b2c
**Internal Communication** Will notify interested parties once deployed

- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a
